docker-compose.yml

```diff
@@ -23,10 +21,6 @@ services:
     restart: unless-stopped
     networks:
       - saleor-backend-tier
-    volumes:
-      - saleor-db:/var/lib/postgresql
-    ports:
-      - 5432:5432
     environment:
       - POSTGRES_USER=saleor
```

Comment:
Maybe moving these also to override would not be a bad idea, since these are dev env variables.